src/components/transaction-list/TransactionList.js

```diff
@@ -209,7 +209,9 @@ export default function TransactionList({
 
       const isSent =
         status === TransactionStatusTypes.sending ||
-        status === TransactionStatusTypes.sent;
+        status === TransactionStatusTypes.sent ||
```

Comment:
Seems like this added logic isn't duplicated in TransactionCoinRow.js line 93 - also, do we care about including TransactionStatusTypes.cancelled in this list as well?

Reply:
Good catch. I was meant to remove it since it's the logic is now handled by `isOutgoing` and `canBeResubmitted`